src/CoreText/Adapter.cs

```diff
@@ -116,6 +116,19 @@ public static string GetStringValue (IDictionary<NSObject, NSObject> dictionary,
 			return ((NSNumber) value).UInt32Value;
 		}
 
+		public static float? GetFloatValue (IDictionary<NSObject, NSObject> dictionary, NSObject key)
```

Comment:
There's already `GetSingleValue` which does the same thing.

Reply:
Ahh woow completely missed that, thanks Rolf!